src/main/java/com/pestphp/pest/types/ThisTypeProvider.java

```diff
@@ -0,0 +1,61 @@
+package com.pestphp.pest.types;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.util.PsiTreeUtil;
+import com.jetbrains.php.lang.psi.elements.*;
+import com.jetbrains.php.lang.psi.elements.impl.FunctionReferenceImpl;
+import com.jetbrains.php.lang.psi.resolve.types.PhpType;
+import com.jetbrains.php.lang.psi.resolve.types.PhpTypeProvider4;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ThisTypeProvider implements PhpTypeProvider4 {
+    private final Set<String> PEST_TEST_FUNCTION_NAMES = new HashSet<>(Arrays.asList("it", "test"));
+    private final PhpType TEST_CASE_TYPE = new PhpType().add("\\PHPUnit\\Framework\\TestCase");
+
+    @Override
+    public char getKey() {
+        return '\u0221';
```

Comment:
What is this and why this value?

Reply:
Just some key for this provider. It is important if provider has overridden `getBySignature` method.